tests/other/metadce/hello_world_fastcomp_O3_MAIN_MODULE.sent

```diff
@@ -252,6 +252,38 @@ ___cxa_current_primary_exception
 ___cxa_decrement_exception_refcount
 ___cxa_end_catch
 ___cxa_find_matching_catch
+___cxa_find_matching_catch_0
+___cxa_find_matching_catch_1
+___cxa_find_matching_catch_10
+___cxa_find_matching_catch_11
+___cxa_find_matching_catch_12
+___cxa_find_matching_catch_13
+___cxa_find_matching_catch_14
+___cxa_find_matching_catch_15
+___cxa_find_matching_catch_16
+___cxa_find_matching_catch_17
+___cxa_find_matching_catch_18
+___cxa_find_matching_catch_19
+___cxa_find_matching_catch_2
+___cxa_find_matching_catch_20
+___cxa_find_matching_catch_21
+___cxa_find_matching_catch_22
+___cxa_find_matching_catch_23
+___cxa_find_matching_catch_24
+___cxa_find_matching_catch_25
+___cxa_find_matching_catch_26
+___cxa_find_matching_catch_27
+___cxa_find_matching_catch_28
+___cxa_find_matching_catch_29
+___cxa_find_matching_catch_3
+___cxa_find_matching_catch_30
+___cxa_find_matching_catch_31
```

Comment:
why does this go up to 31, if the limit is 10?

Reply:
Forgot to update the test - earlier I had it go up to 31, but that was excessive. Updated.